src/main/pluginsServer.ts

```diff
@@ -134,9 +136,19 @@ export async function startPluginsServer(
         scheduleControl = await startSchedule(hub, piscina)
         jobQueueConsumer = await startJobQueueConsumer(hub, piscina)
 
-        queue = await startQueue(hub, piscina)
+        const queues = await startQueues(hub, piscina)
+
+        // `queue` refers to the ingestion queue. With Celery ingestion, we only
+        // have one queue for plugin jobs and ingestion. With Kafka ingestion, we
+        // use Kafka for events but still start Redis for plugin jobs.
+        // Thus, if Kafka is disabled, we don't need to call anything on
+        // redisQueueForKafkaIngestion, as that will also be the ingestion queue.
+        queue = queues.ingestion
+        redisQueueForKafkaIngestion = config.KAFKA_ENABLED ? queues.redis : null
```

Comment:
re: naming, isn't this `redisQueueForPluginJobs` ?

Reply:
Well that's the mess here. The redis queue is both the ingestion queue and the plugin jobs queue. But when we don't have Kafka enabled, the ingestion queue === the redis queue, so we shouldn't be calling methods on it twice.

When Kafka is enabled, on the other hand, we set this queue, which will like you said only be for plugin jobs. So I wanted the naming to make this explicit but not sure if it helped.

Reply:
Ahhhh. `redisQueueForPluginJobsOnKafka` ? 😂 

Hmmm, thinking a bit more, I think the confusion stems from the two queues being called `ingestion` and `redis`? The former name is "what is the queues fuction" and the latter name is "what type of queue it is".

Since on kafka we have two queues, and only one queue on non-Kafka, how about calling the queues `main` and `auxillary` ? and then the auxillary queue name can be `redisQueueForPluginJobs` ? It becomes clear, thanks to Line 147, that the auxillary queue is only set on `config.KAFKA_ENABLED`.

Reply:
.. This might feel pretty trivial, but I think it's important to get this right: we have a million queues already, and it can get confusing for new-comers to understand what goes where if we don't name things properly.

Apologies for the pedantry 😅 .

Reply:
Not a problem at all! That's what reviewers are for. Thanks for upholding this